addons/tls13-rollout-bug1442042/bootstrap.js

```diff
@@ -0,0 +1,61 @@
+/* -*- indent-tabs-mode: nil; js-indent-level: 2 -*- */
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+"use strict";
+
+let {classes: Cc, interfaces: Ci, utils: Cu} = Components;
+
+Cu.import("resource://gre/modules/Preferences.jsm");
+Cu.import("resource://gre/modules/ClientID.jsm");
+
+Cu.importGlobalProperties(["crypto", "TextEncoder"]);
+
+// initially roll out to 10%, we want to control this on the client rather than 
+// depending on server-side throttling, as throttling cannot account for any
+// other concurrent gradual roll-outs.
+const ENABLE_PROB = 0.9;
```

Comment:
talked on IRC but either this should be 0.1 or it should be DISABLE_PROB and the logic below should change